### ISSUE TITLE
TRUE COLORS support

### ISSUE THREAD

**Issue by alberti42** [trapd00r/LS_COLORS#220]
Thank you for the wonderful database of dir colors.

I wrote a python script that converts LS_COLORS to true colors. The script is automatically processed by GitHub actions. For now, I just made a [fork](https://github.com/alberti42/LS_COLORS_TRUE_COLORS) of the repo.

My main motivation is that I wanted the colors to look the same regardless of the terminal emulator — which now happens using true colors.

**Comment by rpdelaney** [comment]
Thanks! Have you seen [vivid](https://github.com/sharkdp/vivid)?

**Comment by alberti42** [comment]
Thank you! I had missed that, but I will try it out.

**Comment by rpdelaney** [comment]
@alberti42 How did that work out for you? I'd still like to do a [merger with vivid](https://github.com/sharkdp/vivid/issues/134) at some point (provided @trapd00r doesn't object). So I'm interested to hear your experiences with it.

**Comment by alberti42** [comment]
@rpdelaney Thanks for launching this initiative!

I remember I spent quite some time on it. I had [forked](https://github.com/alberti42/LS_COLORS_TRUE_COLORS) the repo and made a true color version using my python script for conversion of the color table of trap00r/LS_COLORS. The reason is that I wanted to have consistent colors across terminals without tweaking the terminals on how they interpret the 16 or 256 colors. I believe true colors (24 bit) ensures this in modern terminals.

My zsh script to load trap00r true colors is in https://github.com/alberti42/dotfiles/blob/public/zinit/src/LS_COLORS_TRUE_COLORS/LS_COLORS_TRUE_COLORS.zsh whereas my script for loading vivid colors is https://github.com/alberti42/dotfiles/blob/public/zinit/src/LS_COLORS_TRUE_COLORS/LS_COLORS_TRUE_COLORS.zsh.

At the moment, I am using vivid color scheme `snazzy`. I liked trap00r colors more because they were more detailed and offered a richer set. However, I opted for vivid because it seemed to me to receive better support for the future.

I don't know if this was the kind of feedback you were looking for.

**Comment by rpdelaney** [comment]
Thanks for the feedback :) I guess what I'm trying to feel out is whether merging this repo into a vivid colorscheme would meet your needs as a user, or if it would be necessary to integrate your script into GHA in this repo.

There are other concerns I'm balancing of course, but understanding how people "out there" are using LS_COLORS is a bit opaque to me.

**Comment by alberti42** [comment]
Yes, my take is that it would be great to have the trapd00r color scheme integrated in the vivid package as one of the color schemes.

Probably, you may still want to keep the repo LS_COLORS and just use vivid as a way to publish the color scheme? You may have to give a name to the color scheme in trapd00r github. Does it have an official name? Or just LS_COLORS? It seems to vague and not specific to me to make it into the vivid distribution.
